### PR TITLE
rename master branch to main in 20.21.12 branch

### PR DESCRIPTION
**Pull request by dlockhart**
We need to apply these same changes to the cert branch.

### CODE REVIEW

File: .github/workflows/bsi-update.yml [hunk 1]
```diff
@@ -30,6 +30,5 @@ jobs:
           AUTO_MERGE_TOKEN: ${{ secrets.BSI_AUTO_APPROVE_GITHUB_TOKEN }}
           BRANCH_NAME: ghworkflow/bsi_auto_update
           COMMIT_MESSAGE: 'BSI Auto Update Dependencies'
-          DEFAULT_BRANCH: master
```
Comment (svanherk) [inline .github/workflows/bsi-update.yml:33]: This one shouldn't matter but shouldn't hurt anything

**Review by svanherk — APPROVED** [review]

File: .github/workflows/bsi-update.yml (hunk 1 above)
Comment (dlockhart) [reply .github/workflows/bsi-update.yml:33]: Yeah it was easier to just cherry-pick the whole commit. At least now they're identical!

**Comment by jorge-ramirez-arredondo** [comment]
Is this cherry-picked from another PR or did you just make this changes manually?

**Comment by dlockhart** [comment]
> Is this cherry-picked from another PR or did you just make this changes manually?

Cherry-picked from #5186.

**Pull request merged**